### ISSUE TITLE
display current value (replace %s)

### ISSUE THREAD

**Issue by liweys** [Gericop/Android-Support-Preference-V7-Fix#121]
A small enhancement, which will replace `android:summary="current value is %s"` with current value.

e.g.,
```
public class EditTextPreference extends v7.EditTextPreference {
    ...
    @Override
    public CharSequence getSummary() {
        if(super.getSummary() == null)
            return null;
        String summary = super.getSummary().toString();
        return String.format(summary, getText());
    }
}
```

**Comment by Gericop** [comment]
The `EditTextPreference` has been updated for 27.0.1.1 (to be released soon) to include a summary logic but there's an argument whether it should be in a new class because this behavior differs from the official one and it might be a problem for anyone who wants to use a custom logic.

**Comment by Gericop** [comment]
Use [`AutoSummaryEditTextPreference`](https://github.com/Gericop/Android-Support-Preference-V7-Fix/wiki/Preference-types#autosummaryedittextpreference) to display the current value automatically, released in v**27.0.2.0**.
